ReactViewResources/ReactViewResources.csproj

```diff
@@ -100,4 +93,7 @@
   <PropertyGroup>
     <PreBuildEvent>npm ci</PreBuildEvent>
   </PropertyGroup>
+  <Target Name="GenerateBundle" BeforeTargets="AfterBuild" >
```

Comment:
After build, will this run in the appropriate timming?

Reply:
Updated (depends on CompileTypeScript, and runs as a Build target)